kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/kroxylicious/KroxyliciousOperator.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests.installation.kroxylicious;
+
+import io.kroxylicious.systemtests.resources.operator.SetUpKroxyliciousOperator;
+
+/**
+ * The type Kroxylicious operator.
+ */
+public class KroxyliciousOperator {
+    private final String deploymentNamespace;
+    private SetUpKroxyliciousOperator setUpKroxyliciousOperator;
```

Comment:
I'm not understanding why SetUpKroxyliciousOperator is a separate class.   KroxyliciousOperator with methods like deploy and undeploy makes some sense, but why delegate to another class?